### PR TITLE
Fishing map/layer icons

### PR DESCRIPTION
**Pull request by j8seangel**
Show icon by layer type in:
- [x] Map tooltips
- [x] Timebar tooltips
image
image
image
image
image
image

### CODE REVIEW

**Comment by enriquetuya** [comment]
@j8seangel @satellitestudiodesign loved how this looks, improves clarity a lot. 

Though it is still a bit unclear when you have both encounters for a vessel and the encounters layer. But I need to check again once the tracks are loaded

File: libs/ui-components/src/icon/Icon.tsx [hunk 1]
```diff
@@ -16,13 +16,16 @@ import { ReactComponent as CategoryMarineManager } from '../assets/icons/categor
 import { ReactComponent as CategoryNews } from '../assets/icons/category-news.svg'
 import { ReactComponent as CategoryReports } from '../assets/icons/category-reports.svg'
 import { ReactComponent as Close } from '../assets/icons/close.svg'
+import { ReactComponent as Clusters } from '../assets/icons/clusters.svg'
```
Comment (nerik) [inline libs/ui-components/src/icon/Icon.tsx:19]: IMHO this file deserves to be auto generated from an `ls assets/icons/`. Just an idea, maybe overkill.

**Review by nerik — APPROVED** [review]
This is looking really good, thanks 🥲
Let me know what you think about autogenerating `Icon.tsx`, otherwise looks perfect, will merge.

File: libs/ui-components/src/icon/Icon.tsx (hunk 1 above)
Comment (j8seangel) [reply libs/ui-components/src/icon/Icon.tsx:19]: I like the idea! I would use something like [this](https://github.com/cycloidio/import-index-generator/blob/master/main.js#L44) as base to do it

**Pull request merged**